src/schema/v2/ordered_set.ts

```diff
@@ -12,8 +12,12 @@ import { ResolverContext } from "types/graphql"
 import { artworkConnection } from "./artwork"
 import { connectionFromArraySlice } from "graphql-relay"
 import { getPagingParameters, pageable } from "relay-cursor-paging"
+import { Gravity } from "types/runtime"
 
-const OrderedSetType = new GraphQLObjectType<any, ResolverContext>({
+export const OrderedSetType = new GraphQLObjectType<
+  Gravity.OrderedSet & { cached: number },
+  ResolverContext
+>({
```

Comment:
One other thing we should do--

We'll want the individual items that we return to also be in a connection. I added `itemsConnection` as a start, but we'll need that to support items of different types (only artworks is supported right now). We should extend that to also support `FeaturedLink` (or just... everything).

There was no practical reason to only support artworks before in `itemsConnection` except I was trying to hit a deadline so kept it small in scope!

Reply:
Ah just saw your comment above about the breaking change 😬.

I don't know if we're categorically tied to the `Connection` nomenclature, it's just what we've tended to use (can ask in #graphql for more opinions). It is nice when the return type is explicit.